apluslms_roman/tests/test_cli.py

```diff
@@ -0,0 +1,285 @@
+import sys
+
+from contextlib import contextmanager
+from io import StringIO
+from os import chdir, getcwd, remove
+from os.path import join, exists, isdir
+from shutil import rmtree
+from unittest import mock, TestCase
+
+from apluslms_roman import cli
+
+
+@contextmanager
+def capture_output():
+    new_out, new_err = StringIO(), StringIO()
+    old_out = (sys.stdout, cli.stdout)
+    old_err = (sys.stderr, cli.stderr)
+    try:
+        sys.stdout = cli.stdout = new_out
+        sys.stderr = cli.stderr = new_err
+        yield (sys.stdout), (sys.stderr)
+    finally:
+        sys.stdout, sys.stderr = old_out[0], old_err[0]
+        cli.stdout, cli.stderr = old_out[1], old_err[1]
+
+COURSE = join(getcwd(), 'mock_course')
```

Comment:
`python3 setup.py test` should fail with `getcwd`. You should use `join(dirname(__file__), 'mock_course')`

Reply:
`python3 cli.py test` doesn't work anyway because of the relative imports :cry:

Reply:
It should. At least the test in travis-ci work for current tests in milestone0.3 and feature/new_observer.

Reply:
I tried calling from a few different folders and it didn't work. Maybe moving the test to the root of the project could help? There's at least something there. Will try at some point.

Reply:
Moving them to root folder helped :) Should I move yamlidator tests there, too?

Reply:
Roman client can be executed with `python3 -m apluslms_roman.cli`. I pushed small commit to milestone0.3, which allows to use `python3 -m apluslms_roman`.